Source/Fuse.Controls.Primitives/Image.Visual.uno

```diff
@@ -140,20 +179,22 @@ namespace Fuse.Controls
 		}
 
 		public void Draw(DrawContext dc, Visual element, float2 offset,
-		                 float2 size, float2 uvPosition, float2 uvSize,
-	                     Texture2D tex, ResampleMode resampleMode,
-		                 float4 Color )
+			float2 size, float2 uvPosition, float2 uvSize,
+			float4x4 imageTransform,
+			Texture2D tex, ResampleMode resampleMode,
+			float4 Color )
 		{
 			draw
 			{
 				apply Fuse.Drawing.Planar.Image;
+
 				DrawContext: dc;
 				Visual: element;
 				Size: size;
 				Position: offset;
 				Texture: tex;
 				SamplerState SamplerState: GetSamplerState(resampleMode);
-				TexCoord: VertexData * uvSize + uvPosition;
+				TexCoord: Vector.TransformCoordinate(VertexData * uvSize + uvPosition, imageTransform);
```

Comment:
I think I would do:
```uno
TexCoord: VertexData * uvSize + uvPosition;
TexCoord: Vector.TransformCoordinate(prev, imageTransform);
```
...just to untangle these two steps.

Reply:
Hmm, I kind of expected this to get rid of Fuse.Drawing.Planar.Rectangle.Invert...

Reply:
Do you know when/why Invert is currently used?

Reply:
@kristianhasselknippe: Only when using textures that come from images, AFAIK. It's basically a hard-coding of one of the orientations.

Reply:
Or, the combination of image-orientation and the fact that we're using a confused OpenGL coordinate-system for images.

Reply:
@kusma:
So it turns out Invert is used in a bunch of places like 
Controls.Panels\Panel.Freeze.uno(227) 
Drawing.Surface\Surface.uno(143)
Controls\Viewport.uno(273)
Effects\Mask.uno(104,122,140)
iOS.TextRenderer\TextRenderer.uno(355)

and all of the effect classes. 

Should we perhaps look into getting rid of Invert as a separate issue?